mlrun/feature_store/api.py

```diff
@@ -128,6 +129,35 @@ def get_offline_features(
         print(vector.get_stats_table())
         resp.to_parquet("./out.parquet")
 
+    examples_2::
+
+        names = ['A', 'B', 'C', 'D', 'E']
+        ages = [33, 4, 76, 90, 24]
+        department = ['IT', 'RD', 'RD', 'Marketing', 'IT']
+        data = pd.DataFrame({'name': names, 'age': ages, 'department': department}, index=[0,1,2,3,4])
+        one_hot_encoder_mapping = {
+            "department": list(data["department"].unique()),
+        }
+        data_set = fstore.FeatureSet("fs-new",
+                                     entities=[fstore.Entity("id")],
+                                     description="feature set")
+        data_set.graph.to(OneHotEncoder(mapping=one_hot_encoder_mapping))
+        data_set.set_targets()
+        data_set.plot(rankdir="LR", with_targets=True)
+        df = fstore.ingest(data_set, data,
+                           infer_options=fstore.InferOptions.default())
+
+        fv_name = 'new-fv'
+        features = ['fs-new.name', 'fs-new.age', 'fs-new.department_RD', 'fs-new.department_IT',
+                    'fs-new.department_Marketing']
+        transactions_fv = fstore.FeatureVector(fv_name,
+                                               features,
+                                               description='my feature vector')
+        transactions_fv.save()
+        train_dataset = fstore.get_offline_features(fv_name, target=ParquetTarget(),
+                                                    filter="age>6 and department_RD==1")
```

Comment:
While this example is very elaborate and that's good, it does tend to hide the reason we're here - using the `get_offline_features` function.
I think it would be better to either remove the data-preparation block from this example, or at least provide several examples of using `get_offline_features`. Maybe even use the same fvec shown in the 1st example (which is created from 2 fsets, which is a better example for a fvec) and demonstrate `get_offline_features` with several types of filter strings, and maybe using some combination of the API fields (filtering with `start_time` and `end_time` etc.).